### DISCUSSION TITLE
laser calibration issue

### DISCUSSION THREAD

**Discussion by lemorgan** [NMGRL/pychron#2163] [Q&A] [answered]
I think I'm having trouble with the laser (calibration?) again. I ran a sample yesterday and got the 'invalid laser calib' warning, and then blank levels of gas. I was doing fine with fusions but this is a step heat. log file and screenshot attached. any suggestions? I'm trying to run at 0.8W.

laser calib file:

[PowerOutput]
coefficients = -0.0018,0.6489,0.9313

![Capture](https://user-images.githubusercontent.com/18449986/162776518-c0ebe379-f1fe-476a-b59c-75f86e5993b2.PNG)
[3051-01B-001.log](https://github.com/NMGRL/pychron/files/8465497/3051-01B-001.log)

**Comment by jirhiker** [comment, marked as answer]
given those coefficients for power calibration the minimum requested power is 0.9313.   
```
Watts = -0.0018*x^2+0.6489*x+0.9313
``` 
where `x` is % laser output.   If you request 0.8W then there is no value between 0-100 that satisfies the calibration equation.

**Reply by lemorgan** [reply]
Good point. I've now reset the calibration file to have the 0.9313 value set to 0.2. Hopefully that will let us run at lower values. Will know more in a few hours...

**Reply by lemorgan** [reply]
This seems to have worked - we have gas above blank level after extraction at 0.5W!
